pynance/textimporter_test.py

```diff
@@ -0,0 +1,217 @@
+from __future__ import absolute_import, print_function
+
+import unittest
+import os.path
+
+import numpy as np
+from numpy.testing import assert_array_equal
+
+from .textimporter import CsvImporter, SupportedCsvTypes, \
```

Comment:
Out of curiosity, what is the `.` in `.textimporter` for?

Reply:
the dot indicates "." is a module within this directory, not a package. Python 3 feature, in Python 2 this was ambiguous

Reply:
ok, thx